src/Configuration/JsonSchemaTransformable.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelCom\Integration\Configuration;
+
+use MyParcelCom\Integration\Configuration\Properties\JsonSchemaProperty;
+
+interface JsonSchemaTransformable
```

Comment:
Even though it's an interface, I still think _Form_ is more suitable name. The reason is because you have a class Form, a collection of fields essentially. The fact that a field can be transformed into a json schema property is already covered by the function declared in the interface. If the interface grows with another mandatory method (I am excluding the _isRequired()_ from this example because of my previous comment about it), then what will happen with the interface name?